intro: Describe "Local Additions" folder state and button (#868)
syncthing/docs: intro/gui.rst

Cleaning Versions
	while removing obsolete files from the .stversions folder.

> Local Additions

Reviewer: ```suggestion

Local Additions
```

> 	Indicates receive-only folders with items present which were added or modified locally, thus not synced to remote machines. The red button "Revert Local Changes" will revert and delete all local changes to the state advertised by remote devices.

Reviewer: ```suggestion
	when locally added or modified items are present in a receive-only
	folder, thus not synced to remote machines.  The red button "Revert
	Local Changes" will revert and delete all local changes to the state
	advertised by remote devices.
```
